### ISSUE TITLE
Compability with Cluster API?

### ISSUE THREAD

**Issue by cyang49** [kubernetes-sigs/cluster-api-provider-kubemark#49]
**What steps did you take and what happened:**
[A clear and concise description on how to REPRODUCE the bug.]
I'm trying to use CAPK. I encountered this error

```
$  clusterctl config cluster wow --infrastructure kubemark --flavor capd --kubernetes-version 1.21.1 --control-plane-machine-count=1 --worker-machine-count=4 | kubectl apply -f-
Error: unknown flag: --infrastructure
error: no objects passed to apply
```

The clusterctl I downloaded is 
```
clusterctl version: &version.Info{Major:"1", Minor:"1", GitVersion:"v1.1.3", GitCommit:"31146bd17a220ef6214c4c7a21f1aa57380b6b1f", GitTreeState:"clean", BuildDate:"2022-03-08T18:52:05Z", GoVersion:"go1.17.3", Compiler:"gc", Platform:"linux/amd64"}
```

I saw in the README that the `0.3.0` version CAPK is tested with clusterctl API `0.1.*` But the `0.1.*` version is very old. Is that what I have to use to resolve the compatibility issue? Thank you

**What did you expect to happen:**

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- cluster-api version: 1.1.3
- cluster-api-provider-kubemark version: 0.3.0
- Kubernetes version: (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`):

/kind bug
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api-provider-kubemark/labels?q=area for the list of labels]

**Comment by elmiko** [comment]
thanks for the issue @cyang49 , i created #50 to fix the readme. do you think we need more?

**Comment by cyang49** [comment]
@elmiko see comment [here](https://github.com/kubernetes-sigs/cluster-api-provider-kubemark/pull/50#issuecomment-1137200843)

**Comment by elmiko** [comment]
this was fixed by #50, and should have been closed when it merged.
